tenets/codelingo/go/marshalling/codelingo.yaml

```diff
@@ -20,7 +20,7 @@ tenets:
                 @review comment
                 go.ident:
                   # Replace with your own client/server types (prepended with /resource/).
-                  type == "/resource/tenets/codelingo/go/marshalling/example.go.Server"
+                  type == "/resource/specs/codelingo/go/marshalling/example.go.Server"
```

Comment:
Does this need to be changed back to `tenets` as well? Looks like a file path @Daanikus